Apps/BabylonScripts/draco_decoder_gltf.js

```diff
@@ -0,0 +1,48 @@
+
```

Comment:
We don't want the webassembly version?

Reply:
It doesn't work yet.

Reply:
Okay, we should make a note or file an issue or something. It would be good to avoid adding incomplete work if possible.

Reply:
The goal of VT is to avoid regression. Any test scene that can pass the test must be added. If we only test feature complete PG we wouldn't test anything as xmlhttprequest is still incomplete. Issue is still open for WASM btw.